hosted/gke/p1/p1_import_test.go

```diff
@@ -23,11 +21,15 @@ var _ = Describe("P1Import", func() {
 	})
 
 	AfterEach(func() {
-		if ctx.ClusterCleanup && cluster != nil {
-			err := helper.DeleteGKEHostCluster(cluster, ctx.RancherAdminClient)
-			Expect(err).To(BeNil())
-			err = helper.DeleteGKEClusterOnGCloud(zone, project, clusterName)
+		if ctx.ClusterCleanup {
+			if cluster != nil && cluster.ID != "" {
+				GinkgoLogr.Info(fmt.Sprintf("Cleaning up resource cluster: %s %s", cluster.Name, cluster.ID))
```

Comment:
Can you add this info in other `AfterEach` blocks too?

Reply:
Done